common4j/src/main/com/microsoft/identity/common/java/util/IPlatformUtil.java

```diff
@@ -135,5 +135,5 @@ public interface IPlatformUtil {
      * @return a list of extra query parameters, or null.
      */
     @Nullable
-    List<Map.Entry<String, String>> updateWithAndGetPlatformSpecificExtraQueryParameters(@Nullable List<Map.Entry<String, String>> originalList);
+    List<Map.Entry<String, String>> updateWithAndGetPlatformSpecificExtraQueryParametersForBroker(@Nullable List<Map.Entry<String, String>> originalList);
```

Comment:
If this is for Broker then I'm not sure this needs to go in common repo.

Reply:
Do we actually not ever envision having a need for platform specific query parameters in general (for both Broker and Auth libs)?

Reply:
I would think that there could be a need in the future, but with the passkey feature, there's this additional distinction between Android broker and Android MSAL, where the broker needs to automatically add this qp, and for MSAL, the host apps are responsible for adding the qp/enabling passkeys.

Reply:
It does make sense that some logic which is broker specific should not be in common. So there's definitely an existing issue with having the passkey qp logic in this method (because I don't want this logic to be called from a common or MSAL context)